sql/plsql/PlSqlParser.g4

```diff
@@ -6569,6 +6589,7 @@ non_reserved_keywords_pre12c
     | SWITCHOVER
     | SYNCHRONOUS
     | SYNC
+    | SYS
```

Comment:
I guess it should be either in `non_reserved_keywords_in_12c` or in `non_reserved_keywords_pre12c` but not in both.